### DISCUSSION TITLE
Is there a better description for the `Run Silently` check-box in `RunAutomationStep`?

### DISCUSSION THREAD

**Discussion by Ace-Radom** [BartoszCichecki/LenovoLegionToolkit#1222] [General]
Currently the description is `Execute script or program in background, without creating a new window.`

I must declare that **this description is not wrong**, but may cause misunderstanding for those who has absolute no idea on WINAPI. By checking `Run Silently` check-box, new process will be launched with a [`CREATE_NO_WINDOW`](https://learn.microsoft.com/en-us/windows/win32/procthread/process-creation-flags) creation flag, **AND** this flag is ignored if the application is not a console application (seems like it is not described in the [ProcessStartInfo.CreateNoWindow doc](https://learn.microsoft.com/en-us/dotnet/api/system.diagnostics.processstartinfo.createnowindow?view=net-8.0), but I think they are the same). That means, technically, `Run Silently` check-box only works when the user starts a console program; if the program has a GUI, checking or unchecking the check-box is actually the same. A GUI program will not run in background, even if `Run Silently` is checked.

I tried to write a better description (simple, concise, accurate and easy-understandable) but I'm not able to find one. For now I would just use this description, but I'm wondering if we can find a better one.

**Comment by Ace-Radom** [comment]
@BartoszCichecki would `The process is a console application that is being run without a console window.` be a nice alternative? This sentence comes from the MS document. We could also add `Otherwise it will be ignored`, if necessary.

**Reply by BartoszCichecki** [reply]
Or maybe something like: `Execute console applications, without a console window`?

**Reply by Ace-Radom** [reply]
Yup I think it would also be a good idea.
